releases/release-7.0.0.md

```diff
@@ -0,0 +1,483 @@
+    id INT NOT NULL,
+    fname VARCHAR(30),
+    lname VARCHAR(30),
+    hired DATE NOT NULL DEFAULT '1970-01-01',
+    separated DATE DEFAULT '9999-12-31',
+    job_code INT,
+    store_id INT) PARTITION BY KEY(store_id) PARTITIONS 4;
+    ```
+
+    Starting from v7.0.0, TiDB supports Key partitions and can parse the MySQL `PARTITION BY LINEAR KEY` syntax. However, TiDB ignores the `LINEAR` keyword and uses a non-linear hash algorithm instead. Currently, the `KEY` partition type does not support partition statements with an empty partition column list.
+
+    For more information, see [documentation](/partitioned-table.md#key-partitioning).
+
+### Behavior changes
+
+* TiCDC fixes the issue of incorrect encoding of `FLOAT` data in Avro [#8490](https://github.com/pingcap/tiflow/issues/8490) @[3AceShowHand](https://github.com/3AceShowHand) **tw:ran-huang**
+
+    When upgrading the TiCDC cluster to v7.0.0, if a table replicated using Avro contains the `FLOAT` data type, you need to manually adjust the compatibility policy of Confluent Schema Registry to `None` before upgrading so that the changefeed can successfully update the schema. Otherwise, after upgrading, the changefeed will be unable to update the schema and enter an error state.
+
+* In v7.0.0, the [`LOAD DATA`](/sql-statements/sql-statement-load-data.md) SQL statement adds the `batch_size` parameter to split transactions. The default value of the `batch_size` parameter is `1000`, which means that the data to be imported into TiDB is split into multiple transactions, and each transaction inserts up to 1000 rows of records into TiDB. Prior to v7.0.0, you can use the [`tidb_dml_batch_size`](/system-variables.md#tidb_dml_batch_size) system variable to control the transaction size, but this variable no longer takes effect on `LOAD DATA` starting from v7.0.0.
+
+### System variables
+
+| Variable name  | Change type    | Description |
+|--------|------------------------------|------|
+| `tidb_pessimistic_txn_aggressive_locking` | Deleted | Renamed to [`tidb_pessimistic_txn_fair_locking`](/system-variables.md#tidb_pessimistic_txn_fair_locking-new-in-v700). |
+| [`tidb_enable_non_prepared_plan_cache`](/system-variables.md#tidb_enable_non_prepared_plan_cache) | Modified | Takes effect starting from v7.0.0 and controls whether to enable the [Non-prepared plan cache](/sql-non-prepared-plan-cache.md) feature. |
+| [`tidb_enable_null_aware_anti_join`](/system-variables.md#tidb_enable_null_aware_anti_join-new-in-v630) | Modified | Changes the default value from `OFF` to `ON` after further tests, meaning that TiDB applies Null-Aware Hash Join when Anti Join is generated by subqueries led by special set operators `NOT IN` and `!= ALL` by default. |
+| [`tidb_enable_resource_control`](/system-variables.md#tidb_enable_resource_control-new-in-v660) | Modified | The default value of this variable is changed from `OFF` to `ON`, meaning that the cluster isolates resources by resource group by default. |
```

Comment:
```suggestion
| [`tidb_enable_resource_control`](/system-variables.md#tidb_enable_resource_control-new-in-v660) | Modified | The default value of this variable is changed from `OFF` to `ON`, meaning that the cluster isolates resources by resource group by default. Resource Control is enabled by default in v7.0.0, so that you can use this feature whenever you want. |
```